src/services/EnvironmentVariables.ts

```diff
@@ -50,7 +50,7 @@ export class EnvironmentVariables {
 
 	private readonly YOTICALLBACKURL = process.env.YOTICALLBACKURL;
 
-	private CLIENT_SESSION_TOKEN_TTL_SECS = process.env.CLIENT_SESSION_TOKEN_TTL_SECS;
+	private YOTI_SESSION_TTL_DAYS = process.env.YOTI_SESSION_TTL_DAYS;
```

Comment:
needs to be a number https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Unary_plus

Reply:
Done

Reply:
`private YOTI_SESSION_TTL_DAYS = +process.env.YOTI_SESSION_TTL_DAYS!;`